benches/src/02_replace1k.html

```diff
@@ -35,7 +35,7 @@
 				const elementSelector = 'tr:first-child > td:first-child';
 
 				// MUST BE KEPT IN SYNC WITH WARMUP COUNT IN benches/scripts/config.js
-				const WARMUP_COUNT = 5;
+				const WARMUP_COUNT = 25;
```

Comment:
Per that comment, might be a problem here.

Benchmarks are only ran by the CI if that particular commit contains changes to `**/src/**.js`... that's not great. Will see if I can fix that to skip bogus comments needing to be added & reverted.